orc8r/cloud/go/services/orchestrator/obsidian/models/conversion.go

```diff
@@ -75,6 +75,14 @@ func (m *NetworkFeatures) ToUpdateCriteria(network configurator.Network) (config
 	return GetNetworkConfigUpdateCriteria(network.ID, orc8r.NetworkFeaturesConfig, m), nil
 }
 
+func (m *NetworkSentryConfig) GetFromNetwork(network configurator.Network) interface{} {
```

Comment:
Nit and possibly not worth addressing - but it appears to me that codecov.io says this function, and the ToUpdateCriteria - lack unit test coverage. Is it hard to generate coverage?

Reply:
Also FWIW there is a plugin for this. If you install Sourcegraph plugin for Chrome. There is a sub-feature for sourcegraph plugin in chrome that enables codecov.io coverage highlighting for PRs. See [info here](https://docs.codecov.com/docs/browser-extension).

Reply:
Thanks for pointing this out and suggesting the extension. It's quite helpful. I'll make sure to fully test-cover future PRs. 

It turns out that integrating this new field (SentryConfig) with existing tests of the Network model is non-trivial. I'm now working on another PR that does that. Thanks!